package.json

```diff
@@ -86,7 +86,7 @@
     "jest-environment-jsdom": "^29.7.0",
     "jest-when": "^3.6.0",
     "lint-staged": "^15.2.2",
-    "openmrs": "next",
+    "openmrs": "^5.6.1-pre.1880",
```

Comment:
Ditto

Reply:
<img width="818" alt="Screenshot 2024-06-06 at 13 07 11" src="https://github.com/openmrs/openmrs-form-engine-lib/assets/12844964/57dcdd97-017d-4a92-97f6-5f8a4c17b15b">
When I change these back to next, the CI fails(see screenshot)